### PR TITLE
chore(swap): Convert egress fee amount to egress asset in SwapEgressScheduled handler

### PR DESCRIPTION
**Pull request by niklasnatter**

### CODE REVIEW

**Review by acdibble — APPROVED** [review]

File: packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts [hunk 1]
```diff
@@ -14,17 +15,29 @@ const {
...
-  ),
+  post: jest.fn((url, data) => {
+    if (data.method === 'cf_environment') {
+      return Promise.resolve({ data: environment({ egressFee: '0x777777' }) });
+    }
+
+    if (data.method === 'cf_swap_rate') {
+      return Promise.resolve({
+        data: swapRate({
+          output: `0x${(BigInt(data.params[2]) * 3n).toString(16)}`,
```
Comment (GabrielBuragev) [inline packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts:26]: why `* 3n`?

Comment (niklasnatter) [reply packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts:26]: just a random rate that could be the rate of the pool

**Review by GabrielBuragev — APPROVED** [review]

**Review by jerryafr — APPROVED** [review]

File: packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts [hunk 2]
```diff
@@ -90,4 +103,71 @@ describe(swapEgressScheduled, () => {
       fees: [{ id: expect.any(BigInt), swapId: expect.any(BigInt) }],
     });
   });
+
+  it('updates an existing swap when the egress asset is not the native asset', async () => {
```
Comment (GabrielBuragev) [inline packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts:107]: Can we maybe check if the `getSwapRate` function was called in this test?

Comment (niklasnatter) [reply packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts:107]: the snapshot checks makes sure that the egress fee is three times the fee of the environment 🙂 but will add another assertion in the next pr

**Pull request merged**